src/api/azdevops-api.ts

```diff
@@ -445,20 +443,22 @@ async function createQueryString(additionalFilter: string | undefined, projectUr
     return query;
 }
 
-function analyzeGitRepo(repo: Repository): RepoAnalysis | undefined {
+export function analyzeGitRepo(repo: Repository): RepoAnalysis | undefined {
     if (repo.state.remotes[0].fetchUrl) {
         let remoteRepoName = repo.state.remotes[0].fetchUrl;
         let pathSegments: string[] | undefined;
         let repoSegmentNo = 0;
-        if (remoteRepoName.startsWith("git@")) {  // e.g. git@ssh.dev.azure.com:v3/{org}/{project}/{repo}
-            remoteRepoName = remoteRepoName.substring(remoteRepoName.indexOf("/") + 1);
-            pathSegments = remoteRepoName.split("/");
-            repoSegmentNo = 2;
-        }
-        else if (remoteRepoName.startsWith('https://')) {  // e.g. https://{org}@dev.azure.com/{org}/{project}/_git/{repo}
-            remoteRepoName = remoteRepoName.substring(remoteRepoName.indexOf("/", 10) + 1);
-            pathSegments = remoteRepoName.split("/");
-            repoSegmentNo = 3;
+        if (remoteRepoName.includes("dev.azure.com")) {
```

Comment:
shouldn't we have an else here? I mean, I don't see a lot of other things that "Azure DevOps Simplify" would want to support, but maybe there are still old .visualstudio.com references? And if we check, I think it would be good to at least warn the user that there is something unsupported instead of silently failing. Do you agree?

Reply:
He's not failing silently then. Because then the pathSegments are empty which leads to throwing the error `Couldn't identify the Azure DevOps organization and project from the remote repository fetchUrl xy`.

Reply:
@tfenster is this fine then?

Reply:
not sure how often that is the case, but we still might have very old repos pointing at .visualstudio.com. But let's tackle that when we get there...